python/ray/tune/examples/sigopt_example.py

```diff
@@ -0,0 +1,70 @@
+"""This test checks that SigOpt is functional.
```

Comment:
can you make sure this file is run in the multinode tests? see the other PR for example

Reply:
👍